cmd/samToWig/testdata/test1.sam

```diff
@@ -0,0 +1,7 @@
+@SQ	SN:ref	LN:45
+r001	163	chr1	7	30	8M4I4M1D3M	=	37	39	TTAGATAAAGAGGATACTG	*	XX:B:S,12561,2,20,112
```

Comment:
In the cigar I see 8 matches, 4 matches, and 3 matches.  I think we are incrementing the deleted base as well.  What do we want for the expected behavior here?  Maybe incrementing over the deleted base is the right choice.  Could be a quick discussion to see what everyone thinks.